client/components/Events/ManageEventTemplatesModal.tsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react/no-multi-comp */
+
+import {IFormGroup, IBaseRestApiResponse, IGenericListPageComponent} from 'superdesk-api';
+import {superdeskApi} from '../../superdeskApi';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Modal} from '../index';
+
+interface IProps {
+    handleHide(): void;
+}
+
+interface IEventTemplate extends IBaseRestApiResponse {
+    template_name: string;
+}
+
+export class ManageEventTemplatesModal extends React.PureComponent<IProps> {
+    static propTypes: any;
+
+    render() {
+        const {handleHide} = this.props;
+
+        const {getGenericListPageComponent, ListItemColumn, ListItem} = superdeskApi.components;
+        const {getFormFieldPreviewComponent, FormFieldType} = superdeskApi.forms;
+
+        const EventTemplatesComponent = getGenericListPageComponent<IEventTemplate>('events_template');
```

Comment:
This component does not auto-close the sort options dropdown after making a selection.